examples/card_table_example.py

```
@@ -0,0 +1,153 @@
+        "method": method,
+        "params": params or {}
+    }
+    
+    # Run the MCP server with the request
+    process = subprocess.Popen(
+        [sys.executable, "mcp_server_cli.py"],
+        stdin=subprocess.PIPE,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True
+    )
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)